chart/env/prod.yaml

```diff
@@ -352,6 +358,11 @@ api:
   replicas: 12
   service:
     type: NodePort
+  ingress:
+    enabled: true
+    annotations:
+      alb.ingress.kubernetes.io/group.order: "2"
```

Comment:
as api ingress will take everything at the root, we should put it as the last group, so: 5, and move the other ones.

same for staging

Reply:
an alternative is to explicitly list all the api endpoint, but there are many

Reply:
Removed the annotation. Will use the default group.order (100)